### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by EECOLOR**
Updated dependencies.

As a result of changes in the Javascript world we had to ditch the 'postcss-modules' plugin and replace it with custom code. We also needed to replace 'postcss-next' with 'postcss-preset-env'.

More details in #151

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -17,8 +17,9 @@
     "@kaliber/build": "*",
     "express-basic-auth": "^1.1.4",
     "firebase": "^5.6.0",
-    "firebase-admin": "^6.3.0",
-    "gsap": "^2.0.1",
+    "firebase-admin": "^7.0.0",
+    "gsap": "^2.0.2",
+    "icss-utils": "^4.0.0",
```
Comment (EECOLOR) [inline example/package.json]: dependency should not be here

File: library/webpack-loaders/css-loader.js [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
 const isProduction = process.env.NODE_ENV === 'production'
 
-function createPlugins(loaderOptions, { onExport, resolve, processUrl }) {
+function createPlugins(loaderOptions, { resolve, processUrl }) {
   const { minifyOnly = false, globalScopeBehaviour = false } = loaderOptions
 
   return [
-    // these plugis need to run on each file individual file
-    // look at the source of postcss-modules to see that it effectively runs all modules twice
-    !minifyOnly && require('../postcss-plugins/postcss-plugin-composition')([
```
Comment (EECOLOR) [inline library/webpack-loaders/css-loader.js]: We can remove this plugin

**Comment by klaasman** [comment]
repeatedly imported files (like `/theme.css`) are all separately included in the end result.

**Comment by klaasman** [comment]
production builds are building and running server-side, though on the client there's an error:

```js
TypeError: Cannot set property 'lastEffect' of null
```

**Comment by EECOLOR** [comment]
> repeatedly imported files (like `/theme.css`) are all separately included in the end result.

That is already happening

**Comment by EECOLOR** [comment]
Anoying thing for development:

```
:root {
   ...
}
```

These blocks are copied in the end result. The variables were previously replaced preventing the same behavior.

**Comment by EECOLOR** [comment]
> `TypeError: Cannot set property 'lastEffect' of null`

This should be solved now (with Terser instead of Uglify)

**Comment by EECOLOR** [comment]
The `:root` thing has been fixed (well, worked around). The behavior is now similar to a previous situation.

File: library/webpack-loaders/css-loader.js [hunk 3]
```diff
@@ -1,28 +1,47 @@
...
 const { relative, dirname } = require('path')
+const genericNames = require('generic-names')
 
 const isProduction = process.env.NODE_ENV === 'production'
 
-function createPlugins(loaderOptions, { onExport, resolve, processUrl }) {
+function createPlugins(loaderOptions, { resolve, processUrl }) {
   const { minifyOnly = false, globalScopeBehaviour = false } = loaderOptions
 
   return [
```
Comment (EECOLOR) [inline library/webpack-loaders/css-loader.js]: If we have no plugins to run, we can just skip executing postcss

**Comment by EECOLOR** [comment]
`color(` functions should be replaced with `color-mod(`

**Pull request merged**